WARNING:root:You are using evm version 1.8.2. The supported version is 1.7.3
WARNING:root:You are using solc version 0.4.21, The latest supported version is 0.4.19
INFO:root:contract contracts/39662.sol:ElectricQueue:
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM Code Coverage: 			 34.1%
INFO:symExec:	  Integer Underflow: 			 True
INFO:symExec:	  Integer Overflow: 			 True
Traceback (most recent call last):
  File "oyente.py", line 221, in <module>
    main()
  File "oyente.py", line 216, in main
    exit_code = analyze_solidity()
  File "oyente.py", line 109, in analyze_solidity
    results, exit_code = run_solidity_analysis(inputs)
  File "oyente.py", line 86, in run_solidity_analysis
    result, return_code = symExec.run(disasm_file=inp['disasm_file'], source_map=inp['source_map'], source_file=inp['source'])
  File "/oyente/oyente/symExec.py", line 2458, in run
    ret = detect_vulnerabilities()
  File "/oyente/oyente/symExec.py", line 2277, in detect_vulnerabilities
    detect_parity_multisig_bug_2()
  File "/oyente/oyente/symExec.py", line 2154, in detect_parity_multisig_bug_2
    parity_multisig_bug_2 = ParityMultisigBug2(g_src_map)
  File "/oyente/oyente/vulnerability.py", line 193, in __init__
    self.pairs = self._get_contracts_containing_selfdestruct_opcode()
  File "/oyente/oyente/vulnerability.py", line 222, in _get_contracts_containing_selfdestruct_opcode
    disasm_data = open(pair[0] + ".evm.disasm").read()
IOError: [Errno 2] No such file or directory: u'contracts/39662.sol:IElectricCharger.evm.disasm'
